pvlib/pvsystem.py

```diff
@@ -2334,6 +2355,35 @@ def pvwatts_losses(soiling=2, shading=3, snow=0, mismatch=2, wiring=2,
     return losses
 
 
+def pvsyst_dc_losses_eq_ohms(V_mp_ref, I_mp_ref, dc_ohmic_percent=0,
```

Comment:
I think we can drop 'pvsyst' from the name and credit Pvsyst in a reference. Or, if we keep 'pvsyst' I'd prefer that the source (pvsyst) be towards the end of the name. Some suggestions:

eq_ohms_pvsyst (do we need to categorize the function as dc_losses?)
dc_loss_eq_ohms_pvsyst
dc_loss_pvsyst_eq_ohms (if we expect a collection of Pvsyst loss functions)

Some style/format edits are needed:
- don't capitalize the parameters V_mp_ref and I_mp_ref
- add a Returns block
- %_loss_at_stc won't display as intended. Suggest adding a symbol L_{stc} and defining it in the text.

Reply:
`ohmic_loss_to_equivalent_resistance`?

Reply:
shouldn't `dc_ohmic_percent` be a required parameter for the function?

Reply:
I do think we can drop pvsyst from the name and add it as a reference. maybe `dc_ohms_from_percent`? tough to balance descriptiveness with conciseness for this one. @cwhanse I incorporated your comment into the latest commit. Thanks.

Reply:
> shouldn't dc_ohmic_percent be a required parameter for the function?

I was trying to anticipate that a user might choose PVsyst losses for a `ModelChain` that would set all the PVsyst-style loss functions but not have explicit losses set for each loss. In that case they would not need to have `dc_ohmic_percent` defined and the function could still run but but defaulted to 0.

Reply:
> don't capitalize the parameters V_mp_ref and I_mp_ref

Actually, I just realized that because V_mp_ref and I_mp_ref are coming from `PVSystem.module_parameters` where they are keyed with capitals, then `_build_kwargs` will key them into the `kwargs` with capitals and pass them into the function that way, that is why left them capitalized as parameters. This is similar to the parameters for `calcparams_cec`, some of which remain capitalized. Stylistically, do you prefer to keep everything lower case for the function parameters and force the `kwargs` into lower-cased keys after `_build_kwargs`? I think that would clash with other methods in the module, so I might go back to having these remain capitalized.

Reply:
> I was trying to anticipate that a user might choose PVsyst losses for a ModelChain that would set all the PVsyst-style loss functions but not have explicit losses set for each loss. In that case they would not need to have dc_ohmic_percent defined and the function could still run but but defaulted to 0.

That kind of decision is best left for the PVSystem/ModelChain layer. All of the base functions should make sense on their own.